app/views/maps/index.html.erb

```diff
@@ -2,7 +2,6 @@
 <%= stylesheet_link_tag "/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.css" %>
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.js') %>
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/windRoseLayer.js') %>
-<%= javascript_include_tag('/lib/leaflet-fullhash/leaflet-fullHash.js') %>
```

Comment:
This is `leaflet-fullhash` not `leaflet-hash`

Reply:
Ohh, sorry for this @sashadev-sky, I'll fix it. :astonished: